src/components/Footer.vue

```diff
@@ -1,12 +1,18 @@
+<style>
+.white--text:before {
+  background: none;
+}
+</style>
+
 <template lang="pug">
   v-card.elevation-24
     v-footer(color="basic" padless)
       v-row(justify="center" no-gutters)
-        v-btn.py-2(v-for="link in links" :key="link" :href="link.href" color="white" icon)
-          v-icon {{link.icon}}
-      v-col.py-0.text-center(cols="12")
-       v-btn.py-0.text-center.white--text.font-weight-bold(to='./about' text='' plain='' justify='center' color='white' @click="viewform" ) Contact Us
-      v-col.py-0.text-center.white--text.font-weight-bold(cols="12") {{ new Date().getFullYear() }} — Devlup Labs
+        v-btn.mt-1.px-6(v-for="link in links" :key="link" :href="link.href" color="white" icon)
+          v-icon(size='30') {{link.icon}}
+      v-col.mt-n2.text-center(cols="12")
+       v-btn.text-center.white--text.font-weight-bold(to='./about' text='' plain='' justify='center' color='white' @click="viewform" ) Contact Us
```

Comment:
Style the button so that it actually looks like a button

Reply:
![image](https://user-images.githubusercontent.com/107100879/194708499-6346de85-4982-43c0-88bd-a12e3067a364.png)

is it fine now

Reply:
![image](https://user-images.githubusercontent.com/75736709/194708944-5ab50a99-5b31-404e-96f6-95b5033d17ce.png)
make it something like this

Reply:
![image](https://user-images.githubusercontent.com/107100879/194712464-3d683642-6ea5-4646-8926-4bbba14fe462.png)
is it fine now

Reply:
Umm just make the button exactly as I suggested

Reply:
![image](https://user-images.githubusercontent.com/107100879/194715281-be2337f5-bd54-4659-a562-a41da8ad0e8f.png)
is that ok